dev-support/release-vm/puppet/production/data/nodes/rmvm.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+classes:
+  - docker
+  - packages::manage
+  - ssh
+
+packages::latest:
+  - curl
+  - git
+  - gnupg
+  - gnupg-agent
+  - maven
```

Comment:
Actually, I don't think we need any of `maven`, `openjdk-8-jdk-headless`, `openjdk-11-jdk-headless` because all these tools are provided by the docker environment. Is there a reason to keep these when the whole point of the VM is to establish a reasonable docker environment?

Reply:
+1 on stripping them.